deck-o-cards.js

```diff
@@ -47,3 +64,9 @@ function shuffle(array) {
 
     return array;
 }
+
+topCard(deck_o_cards());
```

Comment:
The function you created accepts an array, here you are passing it a function. What is the output?